yrs/src/block_store.rs

```diff
@@ -173,11 +173,16 @@ impl BlockStore {
         }
     }
     #[inline(always)]
+    pub fn get_block(&self, ptr: &block::BlockPtr) -> &block::Block {
+        &self.clients[&ptr.id.client].list[ptr.pivot as usize]
+    }
+    #[inline(always)]
```

Comment:
In general, I would put inline directives at the very end of the development process. LLVM can do real miracles on it's own and forcing some habits on it can sometimes screw things up instead of making the faster eg. if inlined function causes function calling it to be too big to be inlined of contains some logic that prevents inlining (we already noticed this behavior in .NET where throw exception prevented callee method from being inlined... it was literally faster to wrap exceptions into methods marked to never be inlined).

Reply:
I added these comments when I did initial benchmarks using the BlockStore approach. It improved performance by 10% just because the test did only lookups. I think you are right that we should remove these comments and add them later based on real benchmarks.